python/kserve/kserve/protocol/rest/server.py

```diff
@@ -86,67 +102,159 @@ def create_application(self) -> FastAPI:
+                    v2_endpoints.ready,
+                    response_model=ServerReadyResponse,
+                    tags=["V2"],
+                ),
+                FastAPIRoute(
+                    r"/v2/models",
+                    v2_endpoints.models,
+                    response_model=ListModelsResponse,
+                    tags=["V2"],
+                ),
+                FastAPIRoute(
+                    r"/v2/models/{model_name}",
+                    v2_endpoints.model_metadata,
+                    response_model=ModelMetadataResponse,
+                    tags=["V2"],
+                ),
+                FastAPIRoute(
+                    r"/v2/models/{model_name}/versions/{model_version}",
+                    v2_endpoints.model_metadata,
+                    tags=["V2"],
+                    include_in_schema=False,
+                ),
+                FastAPIRoute(
+                    r"/v2/models/{model_name}/ready",
+                    v2_endpoints.model_ready,
+                    response_model=ModelReadyResponse,
+                    tags=["V2"],
+                ),
+                FastAPIRoute(
+                    r"v2/models/{model_name}/versions/{model_version}/ready",
+                    v2_endpoints.model_ready,
+                    response_model=ModelReadyResponse,
+                    tags=["V2"],
+                ),
+                FastAPIRoute(
+                    r"/v2/models/{model_name}/infer",
+                    v2_endpoints.infer,
+                    methods=["POST"],
+                    response_model=InferenceResponse,
+                    tags=["V2"],
+                ),
+                FastAPIRoute(
+                    r"/v2/models/{model_name}/generate",
+                    v2_endpoints.generate,
+                    methods=["POST"],
+                    tags=["V2"],
+                ),
+                FastAPIRoute(
+                    r"/v2/models/{model_name}/generate_stream",
+                    v2_endpoints.generate_stream,
+                    methods=["POST"],
+                    tags=["V2"],
+                ),
+                FastAPIRoute(
+                    r"/v2/models/{model_name}/versions/{model_version}/infer",
+                    v2_endpoints.infer,
+                    methods=["POST"],
+                    tags=["V2"],
+                    include_in_schema=False,
+                ),
+                FastAPIRoute(
+                    r"/v2/repository/models/{model_name}/load",
+                    v2_endpoints.load,
+                    methods=["POST"],
+                    tags=["V2"],
+                ),
+                FastAPIRoute(
+                    r"/v2/repository/models/{model_name}/unload",
+                    v2_endpoints.unload,
+                    methods=["POST"],
+                    tags=["V2"],
+                ),
+                # OpenAI Protocol
+                FastAPIRoute(
+                    r"/v1/chat/completions",
```

Comment:
I think we discussed to use `/openai/v1/chat/completions` and `/openai/v1/completions`